catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -228,6 +231,22 @@
   private Set<String> requiredAttributes = Collections.emptySet();
   private Map<String, Set<String>> attributeEnumMap = Collections.emptyMap();
 
+  private Map<String, String> keywords;
```

Comment:
✏️ You might want to make this reference `volatile` since it could be accessed from different threads